spec/rspec/core/source/syntax_highlighter_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 class RSpec::Core::Source
   RSpec.describe SyntaxHighlighter do
-    let(:config)      { RSpec::Core::Configuration.new.tap { |c| c.color = true } }
+    let(:config)      { RSpec::Core::Configuration.new.tap { |config| config.no_color = false } }
```

Comment:
Yes, it overrides `spec_helper.rb:13` where I set `config.no_color = true`. This setting preserves the previous assumption that the suite had that color was disabled by default. And I think that's the right default to continue to have:  expectations against plain text are more readable. For example, notifications_spec.rb:232 expects "1.1) Failure/Error", not "1.1) \e[31mFailure/Error"—which is what it gets when color is enabled. Also, I'm not 100% sure but I would assume there would be a way to run specs in or out of a TTY, and we don't want the specs' color settings to vary based on that.

This file needs color enabled, so, since `config.no_color = true` is set in the `spec_helper`, I need to unset it here. Otherwise, for example, the expectation to `be_highlighted` on line 14 fails.